unlock-app/src/components/interface/locks/Manage/elements/WithdrawFundModal.tsx

```diff
@@ -220,18 +220,18 @@ export const WithdrawFundModal = ({
               />
 
               <Input
-                label={`Balance to transfer: ${amountToTransfer} ${symbol}`}
+                label={`Balance to transfer: ${amountToTransfer} of ${balance} ${symbol}`}
                 size="small"
-                type="range"
+                type="number"
```

Comment:
Change this to text and add a number pattern to input.

Set valueAsNumber to true in register options.

Reply:
Please explain so I learn why!

Reply:
Will replace across the board in another PR!